tools/nni_cmd/nnictl_utils.py

```diff
@@ -373,6 +379,153 @@ def webui_url(args):
     nni_config = Config(get_config_filename(args))
     print_normal('{0} {1}'.format('Web UI url:', ' '.join(nni_config.get_config('webuiUrl'))))
 
+def local_clean(directory):
```

Comment:
This kind of code should be in corresponding training service implementation code.

Reply:
nnictl cleanup will not start a restful server to call trainingService.